src/components/text.tsx

```diff
@@ -1,34 +1,26 @@
-import React, { ReactElement } from "react"
+import React, { HTMLAttributes } from "react"
 
-interface TextProps {
+interface TextProps extends HTMLAttributes<HTMLHeadingElement> {
+  children: string | any
+  className?: string
   as?:
     | "h1"
     | "h2"
-    | "p"
+    | "h2about"
+    | "h2episode"
     | "h3"
     | "h3Link"
-    | "h2about"
+    | "p"
     | "pAbout"
-    | "pAuthor"
-    | "h2episode"
     | "pEpisode"
-
-  children: string | any
 }
 
-export default function Text({
-  children,
-  as,
-
-  ...props
-}: TextProps) {
-  let classNamesVariant = ""
-
+export default function Text({ children, as, className, ...props }: TextProps) {
```

Comment:
i think we should add `className = ''` to avoid having `undefined` in the render

![image](https://user-images.githubusercontent.com/360936/146938144-1b286615-6411-4294-8552-5c4f245e1700.png)